server/loginserver.ts

```diff
@@ -149,18 +139,16 @@ class LoginServerInstance {
 			'&servertoken=' + encodeURIComponent(Config.servertoken) +
 			'&nocache=' + new Date().getTime() +
 			'&json=' + encodeURIComponent(JSON.stringify(dataList)) + '\n';
-		/** @type {any} */
 		let requestOptions = url.parse(this.uri + 'action.php');
 		requestOptions.method = 'post';
 		requestOptions.headers = {
 			'Content-Type': 'application/x-www-form-urlencoded',
 			'Content-Length': postData.length,
 		};
 
-		/** @type {any} */
-		let response = null;
-		// @ts-ignore
-		let req = http.request(requestOptions, res => {
+		let response = <null | string | object> null;
```

Comment:
```suggestion
		let response: string | object | null = null;
```

But `AnyObject` might be better